source/router.ts

```diff
@@ -263,7 +268,14 @@ export class Router extends EventTarget {
 			constructedRoute.clientRoute.parent = parent && parent.clientRoute;
 			constructedRoute.clientRoute.component = constructedRoute.component;
 
-			this.constructedRoutes.push(constructedRoute);
+			// replace root route with default child route
+			const duplicateIndex = this.constructedRoutes.findIndex(route => route.path + '' == constructedRoute.path + '');
```

Comment:
use backticks instead of + '', or better toString

Reply:
to string doesn't exist on regex. but i could use backticks if it matters